src/clj/game/cards/resources.clj

```diff
@@ -1973,6 +1974,18 @@
     :leave-play (req (remove-watch state :theophilius-bagbiter)
                      (swap! state assoc-in [:runner :hand-size :base] 5))}
 
+   "Thunder Art Gallery"
+   (let [ability {:once :per-turn
```

Comment:
This card is not once-per-turn, it's first-each-turn. You'll need to use `first-event?` for this.